presto-main/src/main/java/com/facebook/presto/execution/Output.java

```diff
@@ -29,16 +30,19 @@
     private final ConnectorId connectorId;
     private final String schema;
     private final String table;
+    private final ConnectorCommitHandle connectorCommitHandle;
 
     @JsonCreator
     public Output(
             @JsonProperty("connectorId") ConnectorId connectorId,
             @JsonProperty("schema") String schema,
-            @JsonProperty("table") String table)
+            @JsonProperty("table") String table,
+            @JsonProperty("connectorCommitHandle") ConnectorCommitHandle connectorCommitHandle)
```

Comment:
@rschlussel, maven complains about the ConnectorCommitHandle is an abstract type here. Do you know how to use Json here? Or what is the best practice?

Reply:
We can't use `connectorCommitHandle` here as the engine has no idea what the connector impl would look like. Can we use String or Object instead? The String could be a serialized result you wanna send out as the event payload in the end.